src/nvim/ui.c

```diff
@@ -548,3 +555,31 @@ static void ui_mode_change(void)
   conceal_check_cursur_line();
 }
 
+bool ui_is_widget_external(int widget)
+{
+  switch (widget) {
+    case kUITabline:
+      return tabline_external;
+    case kUICmdline:
+      return cmdline_external;
+    case kUIWildmenu:
+      return wildmenu_external;
+    default:
+      return false;
```

Comment:
Leave out the default case. Our build settings will catch a missing case at compile-time.